public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
+
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference to the older version to compare
+            (defaults to HEAD)
+  b_ref     Git reference to the newer version to compare
+            (defaults to the current workspace including changes)
+```
+
+## Description
```

Comment:
we need to start with some motivation for this command - a fast way to compare your current iteration with the previous one as you do some improvements, etc, etc.

Reply:
Good idea. Added a new description intro p now, please check.